{"cert_user_id": "alexandre1985@github", "next_topic_id": 2, "topic": [{"topic_id": 1604066108, "title": "Linux Manjaro can't detect opened 15441 port", "body": "I have UPnP enabled on my router and on my pc firewall I have opened port 15441 both tcp and udp.\r\nBut when I start zeronet with `sudo systemctl start zeronet` and I then do a `sudo systemctl status zeronet` I get this message:\r\n\r\n> ago 25 00:04:26 pc-grande env[6104]: [00:04:26] FileServer Checking port 15441 using portchecker.co...\r\nago 25 00:04:28 pc-grande env[6104]: [00:04:28] FileServer [BAD :(] Port closed: Port 15441 is closed.\r\nago 25 00:04:28 pc-grande env[6104]: [00:04:28] FileServer Trying to open port using UpnpPunch...\r\n\r\nWhen I open ZeroHello I get \"PORT: closed\".\r\nI believe this is a bug on the zeronet daemon", "parent_topic_uri": "1604062980_users_1Cy3ntkN2GN9MH6EaW6eHpi4YoRS2nK5Di", "added": 1503605537, "modified": 1503662193, "source_link": "https://github.com/HelloZeroNet/ZeroNet/issues/1093", "source_type": "github"}], "next_comment_id": 6, "comment": {"1604066108_mirrored_alexandre1985_github": [{"comment_id": 1, "body": "After sometime the zeronet daemon spits out this message from `sudo systemctl status zeronet`\r\n> ago 25 00:04:44 pc-grande env[6104]: [00:04:44] FileServer UpnpPunch run error: UpnpError: Failed to communicate with igd using port 15441 on local machine after 3 tries. in FileServer.py line 74 > UpnpPunch.py line 325 > UpnpPunch.py line 316\r\n\r\nI believe this is relevant", "added": 1503605811, "modified": 1503605811, "source_link": "https://github.com/HelloZeroNet/ZeroNet/issues/1093#issuecomment-324782869", "source_type": "github"}, {"comment_id": 2, "body": "my firewall has port 15441 open, so in there isn't the issue", "added": 1503653148, "modified": 1503653148, "source_link": "https://github.com/HelloZeroNet/ZeroNet/issues/1093#issuecomment-324904538", "source_type": "github"}, {"comment_id": 3, "body": "It says that port is closed. Or my router isn't doing UPnP correctly or it may be blocked by ISP. Am I right?", "added": 1503657213, "modified": 1503657213, "source_link": "https://github.com/HelloZeroNet/ZeroNet/issues/1093#issuecomment-324921385", "source_type": "github"}, {"comment_id": 4, "body": "It is my router. I forwarded the port to my pc and this way everything is being detected as open.\r\nOr my router isn't doing UPnP correctly (which I doubt since I haven't got any issues with other programs) or the UpnpPunch isn't punching the port on my case.\r\nCan you help me resolve this issue? Where can I download Upnppunch to test if it works on my computer?", "added": 1503657748, "modified": 1503657748, "source_link": "https://github.com/HelloZeroNet/ZeroNet/issues/1093#issuecomment-324923758", "source_type": "github"}, {"comment_id": 5, "body": "I have got the same issue on other programs, but I didn't noticed because no one had a port: OPEN\\CLOSE flag. +1 for you.\r\nSo the problem seems to be with my router that has UPnP enable but doesn't seem open ports on demand", "added": 1503662178, "modified": 1503662178, "source_link": "https://github.com/HelloZeroNet/ZeroNet/issues/1093#issuecomment-324944991", "source_type": "github"}]}}